component-test/test_dbus_proxy.py

```diff
@@ -521,7 +521,7 @@ class TestProxyMethods(object):
             "direction": "*",
             "interface": "*",
             "object-path": "*",
-            "method": ["Method1", "Method3", "Method4", "Method2"]
+            "method": ["Method1", "Method3", "Method2", "Method4"]
```

Comment:
Isn't that what the "test_method_order" test is for? I do think this line could be left unchanged because of that test though.